pack/23s.json

```diff
@@ -314,7 +314,7 @@
     "side_code": "corp",
     "stripped_text": "After you resolve this operation, your action phase ends. Play only if the Runner made a run during their last turn. Trace 4. If successful, give the Runner 4 tags.",
```

Comment:
In this stripped text the trace is not formatted with brackets.

Reply:
*this one* was not intentional.  :)